libs/s25main/ingameWindows/iwObservate.cpp

```diff
@@ -143,11 +143,6 @@ void iwObservate::Msg_ButtonClick(const unsigned ctrl_id)
             for(unsigned i = 1; i <= 4; ++i)
                 GetCtrl<ctrlImageButton>(i)->SetPos(
                   DrawPoint(GetCtrl<ctrlImageButton>(i)->GetPos().x - diff, GetSize().y - 50));
-
-            DrawPoint maxPos(VIDEODRIVER.GetRenderSize() - GetSize() - Extent::all(1));
```

Comment:
Why was this removed? This is required if you have the window at right or bottom border and increase its size

Reply:
No? This is already done by `IngameWindow` and simply duplicated here, which breaks the newly introduced behavior.

Reply:
Ah you are right!